database/generateData.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable func-names */
+const faker = require('faker');
+const db = require('./index');
+
+// generate fake data for 100 unique businesses
+const generateBusiness = function () {
+  const businesses = [];
+  for (let i = 0; i < 100; i += 1) {
+    const entry = {
+      name: faker.company.companyName(),
+      claimed: Math.floor(Math.random() * Math.floor(2)),
```

Comment:
Should not need to call `Math.floor(2))`, seems redundant as it will just equal 2.
Same with remaining usage of `Math.floor` on integers below.
```suggestion
      claimed: Math.floor(Math.random() * 2),
```

Reply:
agree with bill about the  Math.floor(), same thing for line 13-16, I feel like there is no need for the extra math.floor